### ISSUE TITLE
Unable to run automation script using Calabash Xcode 7

### ISSUE THREAD

**Issue by saqibkhan124** [calabash/calabash-ios#871]
I am trying to run the sample 'my_first.feature' but i am facing a problem. I am following this tutorial: codingtricky - calabash-ing-ios-applications.

And I also tried different tutorial but unable to figure out why I am getting this error. I am new in iOS-Application automation.

I have also tried this cucumber commands on other projects, there it is working fine but it is not working in my live project. For me application launch and disappears and after some attempts it gives the below error

````
Unable to start. Make sure you've set APP_BUNDLE_PATH to a build supported by this simulator version
  Calabash::Cucumber::Launcher::StartError: 
  
  "Timed out waiting for UIAutomation run-loop Error while writing to fifo. RunLoop::Fifo::NoReaderConfiguredError.
  
  Logfile: /var/folders/l1/2s2v4t051fxggjhtc4v66jwr0000gn/T/run_loop20151011-3665-179p4kf/run_loop.out
  
  2015-10-11 15:50:34.468 instruments[4675:1106729] Attempting to change eventInstruments Trace Complete (Duration : 2.391299s; Output : /var/folders/l1/2s2v4t051fxggjhtc4v66jwr0000gn/T/run_loop20151011-3665-179p4kf/trace.trace)
  
  
   (Calabash::Cucumber::Launcher::StartError)
  /Users/People/.rvm/gems/ruby-2.2.1/gems/calabash-cucumber-0.16.4/lib/calabash-cucumber/launcher.rb:778:in `new_run_loop'
  /Users/People/.rvm/gems/ruby-2.2.1/gems/calabash-cucumber-0.16.4/lib/calabash-cucumber/launcher.rb:635:in `relaunch'
  /Users/People/Desktop/SVNProject/Emirates_UniversalApp/MyTrips_Redesign/iPHONE/EKiPhone/features/support/01_launch.rb:27:in `Before'
    Given I am on the Welcome Screen                 # features/step_definitions/my_first_steps.rb:1
    Then I swipe left                                # calabash-cucumber-0.16.4/features/step_definitions/calabash_steps.rb:222
    And I wait until I don't see "Please swipe left" # calabash-cucumber-0.16.4/features/step_definitions/calabash_steps.rb:158
    And take picture                                 # calabash-cucumber-0.16.4/features/step_definitions/calabash_steps.rb:217

Failing Scenarios:
cucumber features/my_first.feature:6
```

I have also tried 

```
APP_BUNDLE_PATH=/Users/******/Library/Developer/Xcode/DerivedData/*********/Build/Products/De‌​bug-iphonesimulator/*****-cal.app \  
  DEVICE_TARGET='iPhone 5s - Simulator - iOS 9.0' \
  cucumber 

APP_BUNDLE_PATH=/Users/******/Library/Developer/Xcode/DerivedData/*********/Build/Products/De‌​bug-iphonesimulator/*****-cal.app \
  DEVICE_UDID='*************************'  \
  cucumber
```


I have watch app and also some swift code in my iPhone project. I tried to remove the watch app and comment all the swift code than try to run cucumber command. It start working. but when as soon as i uncomment the swift code and try to run cucumber command, i start facing the same problem again. Can anyone please help me out in figuring out the problem I have also downloaded one code from git which contain both swift and Objective-c and tried to run cucumber command, it is also working for me

**Comment by jmoody** [comment]
@saqibkhan124 Please use GitHub markdown to format your questions.

Did you run `calabash setup` and does your app have a Watch extension?  If so, you are probably experiencing this issue: **`calabash setup` adds calabash.framework to Watch extension instead of the app** #832

**Comment by jmoody** [comment]
Closing for lack of response.  My best guess is that the app that was linked was the _watch extension_.

**Comment by jmoody** [comment]
I have released run_loop 2.0.10.pre1 which might resolve the FIFO problems.  Thanks @TeresaP
